Add type Boolean as SQLValue

```
Caused by: java.io.IOException: Not found type 'Boolean'
	at com.github.smallcreep.cucumber.seeds.table.TableXml.lambda$null$0(TableXml.java:134)
	at org.cactoos.map.MapOf.lambda$new$0(MapOf.java:83)
	at org.cactoos.func.UncheckedFunc.lambda$apply$0(UncheckedFunc.java:56)
	at org.cactoos.scalar.CheckedScalar.value(CheckedScalar.java:76)
	at org.cactoos.scalar.IoCheckedScalar.value(IoCheckedScalar.java:63)
	at org.cactoos.scalar.UncheckedScalar.value(UncheckedScalar.java:56)
	at org.cactoos.func.UncheckedFunc.apply(UncheckedFunc.java:57)
	at org.cactoos.iterator.Mapped.next(Mapped.java:68)
	at org.cactoos.iterator.Immutable.next(Immutable.java:58)
	at org.cactoos.map.MapOf.lambda$new$2(MapOf.java:153)
	at org.cactoos.scalar.CheckedScalar.value(CheckedScalar.java:76)
	at org.cactoos.scalar.IoCheckedScalar.value(IoCheckedScalar.java:63)
	at org.cactoos.scalar.UncheckedScalar.value(UncheckedScalar.java:56)
	at org.cactoos.map.MapEnvelope.keySet(MapEnvelope.java:123)
	at com.github.smallcreep.cucumber.seeds.table.TableSimple.lambda$insert$1(TableSimple.java:92)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at com.github.smallcreep.cucumber.seeds.table.TableSimple.insert(TableSimple.java:91)
	at com.github.smallcreep.cucumber.seeds.table.TableXml.insert(TableXml.java:114)
	at com.github.smallcreep.cucumber.seeds.rows.RwDefault.add(RwDefault.java:145)
	at com.github.smallcreep.cucumber.seeds.db.steps.StpDefInsert.insert(StpDefInsert.java:79)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at cucumber.runtime.Utils$1.call(Utils.java:40)
	at cucumber.runtime.Timeout.timeout(Timeout.java:16)
	at cucumber.runtime.Utils.invoke(Utils.java:34)
	at cucumber.runtime.java.JavaStepDefinition.execute(JavaStepDefinition.java:38)
	at cucumber.runtime.StepDefinitionMatch.runStep(StepDefinitionMatch.java:37)
	at cucumber.runtime.ExtendedRuntime.runStep(ExtendedRuntime.java:319)
	at cucumber.runtime.model.StepContainer.runStep(StepContainer.java:44)
	at cucumber.runtime.model.StepContainer.runSteps(StepContainer.java:39)
	at cucumber.runtime.model.CucumberScenario.run(CucumberScenario.java:44)
	at com.github.mkolisnyk.cucumber.runner.ExtendedExecutionUnitRunner.run(ExtendedExecutionUnitRunner.java:32)
	at com.github.mkolisnyk.cucumber.runner.ExtendedFeatureRunner.runChild(ExtendedFeatureRunner.java:81)
	at com.github.mkolisnyk.cucumber.runner.ExtendedFeatureRunner.runChild(ExtendedFeatureRunner.java:23)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at cucumber.runtime.junit.FeatureRunner.run(FeatureRunner.java:70)
	at com.github.mkolisnyk.cucumber.runner.ExtendedCucumber.runChild(ExtendedCucumber.java:108)
	at com.github.mkolisnyk.cucumber.runner.ExtendedCucumber.runChild(ExtendedCucumber.java:32)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at com.github.mkolisnyk.cucumber.runner.ExtendedCucumber.run(ExtendedCucumber.java:148)
	at com.github.mkolisnyk.cucumber.runner.parallel.CucumberRunnerThread.run(CucumberRunnerThread.java:25)
	at java.lang.Thread.run(Thread.java:748)
```

--- a/seeds-db/src/main/java/com/github/smallcreep/cucumber/seeds/sqlvalue/SqlBoolean.java
+++ b/seeds-db/src/main/java/com/github/smallcreep/cucumber/seeds/sqlvalue/SqlBoolean.java
@@ -1,0 +1,43 @@
+/*
+ * MIT License
+ *
+ * Copyright (c) 2018 Ilia Rogozhin (@smallcreep) <ilia.rogozhin@gmail.com>
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package com.github.smallcreep.cucumber.seeds.sqlvalue;
+
+/**
+ * Boolea sql value.
+ * @since 0.2.1
+ */
+public final class SqlBoolean extends SqlValueCastEnvelope {
+
+    /**
+     * Ctor.
+     */
+    public SqlBoolean() {
+        super(
+            "boolean",
+            new SqlValueAsIs() {
+            }
+        );
+    }
+}

--- a/seeds-db/src/main/java/com/github/smallcreep/cucumber/seeds/table/TableXml.java
+++ b/seeds-db/src/main/java/com/github/smallcreep/cucumber/seeds/table/TableXml.java
@@ -28,6 +28,7 @@ import com.github.smallcreep.cucumber.seeds.SqlValue;
 import com.github.smallcreep.cucumber.seeds.Table;
 import com.github.smallcreep.cucumber.seeds.sqlvalue.Bytea;
 import com.github.smallcreep.cucumber.seeds.sqlvalue.Serial;
+import com.github.smallcreep.cucumber.seeds.sqlvalue.SqlBoolean;
 import com.github.smallcreep.cucumber.seeds.sqlvalue.SqlInteger;
 import com.github.smallcreep.cucumber.seeds.sqlvalue.Text;
 import com.github.smallcreep.cucumber.seeds.sqlvalue.Varchar;
@@ -91,6 +92,10 @@ public final class TableXml implements Table {
                 new MapEntry<>(
                     "Integer",
                     new SqlInteger()
+                ),
+                new MapEntry<>(
+                    "Boolean",
+                    new SqlBoolean()
                 )
             )
         );

--- a/seeds-db/src/test/java/com/github/smallcreep/cucumber/seeds/sqlvalue/SqlBooleanTest.java
+++ b/seeds-db/src/test/java/com/github/smallcreep/cucumber/seeds/sqlvalue/SqlBooleanTest.java
@@ -1,0 +1,50 @@
+/*
+ * MIT License
+ *
+ * Copyright (c) 2018 Ilia Rogozhin (@smallcreep) <ilia.rogozhin@gmail.com>
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package com.github.smallcreep.cucumber.seeds.sqlvalue;
+
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.Matchers;
+import org.junit.Test;
+
+/**
+ * Test Case for {@link SqlBoolean}.
+ * @since 0.2.1
+ */
+public final class SqlBooleanTest {
+
+    /**
+     * SqlBoolean can return value envelope to cast boolean.
+     * @throws Exception if fails
+     */
+    @Test
+    public void returnValueEnvelopeToSingleQuotes() throws Exception {
+        MatcherAssert.assertThat(
+            new SqlBoolean().apply("true"),
+            Matchers.equalTo(
+                "CAST(true as boolean)"
+            )
+        );
+    }
+}

